### PR TITLE
move pulse-api into the network-api repo

### PR DESCRIPTION
**Pull request by Pomax**
This puts the pulse-api into the network-api as a parallel set of apps with associated urls, loaded through the networkapi's settings.py and friends.

It switches over the pulseapi's custom user from "an override on the user class" to a plain model "user profile" instead: there is a "user logged in" signal handler that checks whether a user as a pulse-specific user profile when a login occurs, and if not, it builds one for them. This lets us put things like bookmarks in the user profile, not the user itself, which is much cleaner.

URLs are preseved, I did not do any special namespacing there, so other than the primary domain changing, the actual routes to be used stay the same.

This is marked as "WIP" mostly because I had to rip out the tests that we had for verifying user login and posting etc. so I still need to put those back in after rewriting them to fit the "new" login system as far as pulseapi code is concerned.

### CODE REVIEW

File: app/pulseapi/tests.py.unused [hunk 1]
```diff
@@ -0,0 +1,70 @@
+from django.test import TestCase
```
Comment (Pomax) [inline app/pulseapi/tests.py.unused]: This still needs to be re-jiggered

File: app/pulseapi/settings.py.unused [hunk 2]
```diff
@@ -0,0 +1,198 @@
+"""
```
Comment (Pomax) [inline app/pulseapi/settings.py.unused]: I think we can delete this? /cc @cadecairos

Comment (cadecairos) [inline app/pulseapi/settings.py.unused]: yes, I think so

**Comment by cadecairos** [comment]
@pomax I think we should group the Pulse related apps under one header on the left nav - any objections to that?

File: app/networkapi/urls.py [hunk 3]
```diff
@@ -29,9 +29,11 @@
     url(r'^api/people/', include('networkapi.people.urls')),
     url(r'^api/news/', include('networkapi.news.urls')),
     url(r'^$', mezzanine.pages.views.page, {'slug': '/'}, name='home'),
+    url(r'^', include('pulseapi.urls')),
```
Comment (cadecairos) [inline app/networkapi/urls.py]: So looking at this, I feel like long term these urls should go under the `/api` prefix, for consistencies sake.

what if we expose the views on the old routes and on `/api` to give us a window to update things that depend on those routes.

**Comment by Pomax** [comment]
@cadecairos grouping Pulse entries under a single heading makes sense to me. I'll work that into the PR

File: app/networkapi/urls.py (hunk 3 above)
Comment (Pomax) [inline app/networkapi/urls.py]: a great idea.

**Comment by Pomax** [comment]
Filed followup as https://github.com/mozilla/network-api/issues/196

**Pull request merged**